generators/client/templates/vue/package.json.ejs

```diff
@@ -96,18 +96,16 @@ limitations under the License.
     "<%= module.name %>": "<%= module.version %>",
     <%_ });
 } _%>
-    "html-webpack-plugin": "4.3.0",
+    "html-webpack-plugin": "4.5.0",
     <%_ if (!skipCommitHook) { _%>
-    "husky": "4.3.0",
+    "husky": "VERSION_MANAGED_BY_CLIENT_COMMON",
+    "lint-staged": "VERSION_MANAGED_BY_CLIENT_COMMON,
     <%_ } _%>
     "jest": "26.4.2",
     "jest-junit": "11.1.0",
     "jest-serializer-vue": "2.0.2",
     "jest-sonar-reporter": "2.0.0",
     "jest-vue-preprocessor": "1.7.1",
-    <%_ if (!skipCommitHook) { _%>
```

Comment:
I think we should keep this condition

Reply:
It is still there. Just put both deps under one condition:

```
    <%_ if (!skipCommitHook) { _%>
    "husky": "VERSION_MANAGED_BY_CLIENT_COMMON",
    "lint-staged": "VERSION_MANAGED_BY_CLIENT_COMMON,
    <%_ } _%>
```